source/Ship.cpp

```diff
@@ -739,6 +747,10 @@ string Ship::FlightCheck() const
 		return "limited turn?";
 	if(energy - .8 * solar < .2 * (turnEnergy + thrustEnergy))
 		return "solar power?";
+	if(fuel < 0.)
+		return "fuel?";
+	if(!hyperDrive && !jumpDrive && !canBeCarried)
```

Comment:
This appears to be a duplicate check (to the one immediately below it, which was added in #3635 and adjusted in #3843 ).